constants.js

```diff
@@ -86,6 +86,16 @@ if (!abilities.nevermore_shadowraze3)
   // Find and replace short raze range with long raze range
   abilities.nevermore_shadowraze3.attrib = abilities.nevermore_shadowraze3.attrib.replace(/\d{3}/, 700);
 }
+if (!abilities.beastmaster_call_of_the_wild_boar)
+{
+  abilities.beastmaster_call_of_the_wild_boar = Object.assign(
```

Comment:
I'm curious, where did you get the information?

A future plan is to localize the text strings so the site can be viewed in multiple languages.  If we do that, we'll need to know where to get this data in other languages.